ClientinfoPlugin.cs

```diff
@@ -7,27 +7,28 @@ namespace DNWS
 {
   class ClientinfoPlugin : IPlugin
   {
-    protected static Dictionary<String, int> clientinfoDictionary = null;
+    // protected static Dictionary<String, int> clientinfoDictionary = null;
     public ClientinfoPlugin()
     {
-      if (clientinfoDictionary == null)
-      {
-        clientinfoDictionary = new Dictionary<String, int>();
-
-      }
+      // if (clientinfoDictionary == null)
```

Comment:
It's not good to remove code by commenting it out, it's still there. You should remove them completely. We have git to track the changes, so you don't need to worry that you will lose the code.